### DISCUSSION TITLE
input field does not clear

### DISCUSSION THREAD

**Discussion by NiamatUllahUZR** [livewire/livewire#7237] [1. Help]
when i call $this->reset(), it reset all the public property but still showing data in input field, further in the beganing this work fine as i am coding more i am getting this problem in livewire 3. please guid me if you know something about this. Thanks

**Comment by Golveira** [comment]
I have the same problem. I have a button to clear the form but it's not clearing the data of the inputs, only the properties.

**Comment by Ashraam** [comment]
Did you try adding the .live on wire:model to refresh them instantly ?

Can you show us some code ?

**Comment by Golveira** [comment]
I found the problem. In my case, I think it was a bug with laravel breeze and livewire 3. Alpine was being loaded twice since livewire 3 already comes with it built-in. I removed alpine from app.js, and it's working now.

I also found this [https://laraveldaily.com/post/livewire-3-laravel-breeze-error-alpine-js-conflict](https://laraveldaily.com/post/livewire-3-laravel-breeze-error-alpine-js-conflict)

**Reply by VictorBaleroni** [reply]
Era isso mesmo tambem estava tendo o mesmo problema. Obrigado!

**Reply by GeorgeC137** [reply]
Thanks for the link. It was very helpful
